### PR TITLE
fix no issues to predict issue

### PR DESCRIPTION
**Pull request by lanking520**
This is a fix on the recent issue report sent failure. Adding a checking scheme on the issues number

### CODE REVIEW

**Comment by lanking520** [comment]
@marcoabreu @larroy

File: mxnet-bot/PredictLabels/application.py [hunk 1]
```diff
@@ -57,7 +57,9 @@ def predict():
     # data would be a json file {"issues":[1,2,3]}
     data = request.get_json()["issues"]
     # predictions = predict_labels.predict(data)
-    predictions = predictor.predict(data)
+    predictions = []
+    if len(data) != 0:
```
Comment (KellenSunderland) [inline mxnet-bot/PredictLabels/application.py]: is data a list?  Could we just do
```python
if data:
```

Comment (lanking520) [reply mxnet-bot/PredictLabels/application.py]: Yes, the data is a list. I am not familiar with Python. Does `if data` means if data is `not null` or `data isEmpty`?

**Comment by lanking520** [comment]
Can we merge this? @KellenSunderland @marcoabreu

**Pull request merged**